docs/scripting/language/Style.md

```diff
@@ -187,7 +187,8 @@ Functions must be named with `PascalCase`.
 
 #### Global Variables
 
-Global variables must always use `g` prefixed PascalCase, so `gVariableName` and should generally always be `static`.
+Global variables declared using `new` must always use `g_` prefixed PascalCase, so `g_VariableName`,
```

Comment:
I was against this but now I feel the `g_` prefix matches others such as `E_`/`e_` for enum. `s_` for static also makes sense.

Reply:
That's also one of the reason I proposed the changes